### ISSUE TITLE
I get an install error

### ISSUE THREAD

**Issue by noipllort** [LouisAsanaka/PyChessBot#1]
I got the error chess.uci is not found. I tried to pip install chess , and the version in requirements.txt but it did not work. Can you help me, I alerady made a chess engine in python (using the built in library, so it was not impossible) to play how i want.

**Comment by noipllort** [comment]
Never mind, I fixed it!

**Comment by LouisAsanaka** [comment]
Good to hear!

**Comment by noipllort** [comment]
I deleted one of your files, i thought it was working but it didnt work anyway. I already made my own chess bot + using an external engine with subprocess instead of the garbage chess library (i first tried with that i read on the python chess website but it is garbage). It can solve chess puzzles too. (proof video https://www.youtube.com/watch?v=VtVLDqdVVEE&feature=youtu.be) I will upload video about automatic playing vs stockfish level 6.
